### ISSUE TITLE
add support for multiple translations entries mixing LTR and RTL text

### ISSUE THREAD

**Issue by userquin** [elk-zone/elk#2544]
Right now there is no way to include multiple lang texts when composing, the lang in the composer will apply to the full text.

The new proposal is about to support LTR/RTL text using multiple paragraphs using `bdi` or just adding `dir="auto"` (it also works).

For example, if anyone wants to include english and some arabic translation (or viceversa), we can enable a new flag in the composer options to include inside any `p` a new `bdi` child (it is just an span) with the text (rn the text will be just added to the `p`).

<details>
<summary>Here an example using English + Arabic translation using &lt;bdi></summary>
<br/>
<p><bdi>I like how the arabic dual form is supported. The last one is mistranslated and I'm trying to find how to fix it. Also, the form for 0 needs to be fixed.</bdi></p>
<p><bdi>أنا أحب الطريقة التي يتم بها دعم النموذج المزدوج العربي. تمت ترجمة الكلمة الأخيرة بشكل خاطئ وأحاول العثور على كيفية إصلاحها. أيضًا، يجب إصلاح نموذج 0.</bdi></p>
</details>

<details>
<summary>Previous example without &lt;bdi> (gh adding dir="auto")</summary>
<br/>
<p>I like how the arabic dual form is supported. The last one is mistranslated and I'm trying to find how to fix it. Also, the form for 0 needs to be fixed.</p>
<p>أنا أحب الطريقة التي يتم بها دعم النموذج المزدوج العربي. تمت ترجمة الكلمة الأخيرة بشكل خاطئ وأحاول العثور على كيفية إصلاحها. أيضًا، يجب إصلاح نموذج 0.</p>
</details>

Here Elk toot (notice the final dot is wrong, should be after the 0 in RTL):

![imagen](https://github.com/elk-zone/elk/assets/6311119/632b6f57-cdea-4ab3-a0b9-f93d9207715b)

The `content-rich` wrapper will use `lang="en"` in previous example:

![imagen](https://github.com/elk-zone/elk/assets/6311119/981c8e9f-6dd9-4191-aa32-9db9b73180f0)

/cc @patak-dev @danielroe check the source code in previous details and the source page

**Comment by userquin** [comment]
Here another Elk example using Arabic + English translation:

![image](https://github.com/elk-zone/elk/assets/6311119/65149623-f842-4dac-af58-0495a6ef8a48)
